scanner/src/main/kotlin/storages/PostgresStorage.kt

```diff
@@ -191,7 +192,7 @@ class PostgresStorage(
         @Suppress("TooGenericExceptionCaught")
         return try {
             val scanResults = runBlocking(Dispatchers.IO) {
-                packages.chunked(max(packages.size / 50, 1)).map { chunk ->
+                packages.chunked(max(packages.size / LocalScanner.NUM_STORAGE_THREADS, 1)).map { chunk ->
```

Comment:
BTW, do we actually need the `max()`, as we already exit early above when `packages` is empty?

Reply:
Ah, yes, as for `packages.size` < `LocalScanner.NUM_STORAGE_THREADS` we'd also get 0 from the integer division.

Reply:
Yes, if `packages.size < LocalScanner.NUM_STORAGE_THREADS` the chunk size would be `0`.